SpokojDucha/components/SearchDecedent.js

```diff
@@ -64,98 +58,29 @@ const SearchDecedent = () => {
     }
   };
 
-  const handleNavigate = (id) => {
```

Comment:
Anothere delete that never should happen. Not only you deleted the navigation ability but also you deleted the rendering of data that we get from backend